CHANGELOG.md

```diff
@@ -89,6 +91,10 @@
 - Use SCOPED object for internal create temp tables. The SCOPED objects will be stored sproc scoped if created within stored sproc, otherwise will be session scoped, and the object will be automatically cleaned at the end of the scope.
 - Improved warning messages for operations that lead to materialization with inadvertent slowness.
 - Removed unnecessary warning message about `convert_dtype` in `Series.apply`.
+<<<<<<< HEAD
```

Comment:
resolve conflict.
also this chance seems in a change log for a released version, why is that? is that a mistake?